code/obj/machinery/manufacturer.dm

```diff
@@ -562,9 +555,9 @@
 		if(usr.stat || usr.restrained())
 			return
 
-		if(src.electrified != 0)
+		if(src.electrified)
 			if (!(status & NOPOWER || status & BROKEN))
-				if (src.manuf_zap(usr, 10))
+				if (src.shock(usr, 10))
 					return
 
 		if ((usr.contents.Find(src) || ((BOUNDS_DIST(src, usr) == 0 || isAI(usr)) && istype(src.loc, /turf))))
```

Comment:
```suggestion
		if (BOUNDS_DIST(src, usr) == 0 || isAI(usr)) && istype(src.loc, /turf))
```
I don't really care to check for the case where the manufacturer is inside someone (???)

Reply:
cyborg moment, but they wouldn't have one of these anyways